### PR TITLE
Fix regression in Projects.create

### PR DESCRIPTION
**Pull request by richard-dds**
A new project's creator was not being assigned a role in the project's environments.

### CODE REVIEW

**Comment by montana-mil** [comment]
Two things I've noticed: there isn't an owner role in environment roles, and when I go to update a project and add a new environment I get an error. I'm not really sure how the update project works. Seems mysterious.

**Comment by patricksmithdds** [comment]
@richard-dds We may need to discuss this further -- there is no "owner" role in an environment, so adding that would make things inconsistent with the desired CSP roles. 

I was thinking that we can't presume to know what role the user intends for themselves, so we can refrain from adding any env role and allow them to add themselves as desired.

File: atst/domain/environments.py [hunk 1]
```diff
@@ -21,18 +21,20 @@ def create(cls, project, name):
...
+        db.session.add_all(environments)
+        return environments
 
     @classmethod
     def add_member(cls, environment, user, role):
         environment_user = EnvironmentRole(
             user=user, environment=environment, role=role
         )
         db.session.add(environment_user)
-        db.session.commit()
```
[the remark this answers is not in the document]
Comment (richard-dds) [reply atst/domain/environments.py]: In the audit logs story I removed some of the commits from the `Projects.create` code path by only `add`ing the new objects to the session and then committing them at the end of `Projects.create`. The benefits are that it saves us a few trips to the database, and it causes the audit events to be generated in a more readable way.

In doing that, I introduced this bug, but in fixing it I also realized that there were more extraneous `commit()`s to remove.

**Comment by richard-dds** [comment]
@montana-mil @patricksmithdds You're both right about the owner role. The only environment role we have now is `nonsense_role`, since we've punted on defining them.

The reason we've been adding project creators to a new project's environments in the first place is so they can view the project's environments. Now I'm realizing that the workspace owner's `VIEW_ENVIRONMENT_IN_APPLICATION` permission will allow them to see all environments anyway.

**Comment by patricksmithdds** [comment]
> I don't see any issues with not adding anyone to a new project's environments, except maybe the fact that the workspace owner may be listed with no environment access on the members page. How do we feel about that?

I think that's intended behavior -- we won't add a user to the environment unless they do so explicitly.

**Comment by richard-dds** [comment]
@patricksmithdds 👍 Updated

**Pull request merged**